Production/python/Production.py

```diff
@@ -123,13 +123,13 @@
     tauIdEmbedder.runTauID() # note here, that with the official CMSSW version of 'runTauIdMVA' slimmedTaus are hardcoded as input tau collection
     boostedTaus_InputTag = cms.InputTag('slimmedTausBoosted')
 elif isRun3:
-    # from TauMLTools.Production.runTauIdMVA_new import TauIDEmbedder
-    # updatedTauName = "slimmedTausNewIDv2p5"
-    # tauIdEmbedder = TauIDEmbedder(
-    #     process, cms, updatedTauName = updatedTauName,
-    #     toKeep = [ "deepTau2017v2p5" ]
-    # )
-    # tauIdEmbedder.runTauID()
+    from TauMLTools.Production.runTauIdMVA_new import TauIDEmbedder
```

Comment:
DeepTau v2p5 is already available in the latest version of [runTauIdMVA.py](https://github.com/cms-sw/cmssw/blob/master/RecoTauTag/RecoTau/python/tools/runTauIdMVA.py) from CMSSW => use it and remove `runTauIdMVA_new.py`